src/main/java/com/cloudbees/jenkins/plugins/BitBucketTrigger.java

```diff
@@ -156,7 +163,8 @@ public void writeLogTo(XMLOutput out) throws IOException {
 
         @Override
         public boolean isApplicable(Item item) {
-            return item instanceof AbstractProject;
+            return item instanceof Job && SCMTriggerItem.SCMTriggerItems.asSCMTriggerItem(item) != null
+                    && item instanceof ParameterizedJobMixIn.ParameterizedJob;
```

Comment:
I doubt the last condition is required. Non-parameterized jobs can be triggered as well, but they may be unable to access the metadata